Backend/app/provider-platform/dynamic-offer-driver-app/Main/dynamic-offer-driver-app.cabal

```diff
@@ -301,6 +301,8 @@ library
       SharedLogic.FareProduct
       SharedLogic.GoogleMaps
       SharedLogic.GoogleTranslate
+      SharedLogic.Location.Backfill
+      SharedLogic.Location.Location
```

Comment:
Two more modules are not required. Just move all this code inside `SharedLogic.LocationMapping` and rename it to `SharedLogic.Location`